src/test/unit/mcmc/hmc/hamiltonians/softabs_metric_test.cpp

```diff
@@ -59,6 +59,8 @@ TEST(McmcSoftAbs, sample_p) {
   EXPECT_EQ("", fatal.str());
 }
 
+#ifndef STAN_OPENCL
+
```

Comment:
why this change?

Reply:
I added this one because this test also uses fvars and we dont support to_matrix_cl for fvars. The alternative is to only run the assign_cl and rvalue_cl tests.